components/org.wso2.carbon.stream.processor.statistics/src/main/java/org/wso2/carbon/stream/processor/statistics/internal/OperatingSystemMetricSet.java

```diff
@@ -74,12 +74,17 @@ protected void start(BundleContext bundleContext) {
     /**
      * Get the MBean name from the deployment yaml and get access to the MBean.
      */
-    public OperatingSystemMetricSet() {
+    public OperatingSystemMetricSet() {}
+
+public void initConnection(){
+    try {
         metricManagementService = StreamProcessorStatisticDataHolder.getInstance().getMetricsManagementService();
         isJMXEnabled = metricManagementService.isReporterRunning("JMX");
         mBeanServer = ManagementFactory.getPlatformMBeanServer();
+    } catch (IllegalArgumentException e){
+        LOGGER.warn("Worker level jmx reporting has disabled.");
```

Comment:
Add the exception message and exception

Reply:
Fixed in a119095399fb91865c665aa38fb459ff303ab51a